### PR TITLE
Update providers.json

### PR DESCRIPTION
**Pull request by JavituRec**
Added a hosting

### CODE REVIEW

File: _data/providers.json [hunk 1]
```diff
@@ -6,6 +6,11 @@
     "java_ip": "Make sure your remote address is 'auto', uncomment `bedrock address`, and change `0.0.0.0` to your Java server's IP."
   },
   "built_in": [
+    {
+      "name": "GameProHost",
+      "url": "https://gameprohost.com/",
+      "description": "To install Geyser on GameProHost is very simple, you just need to install the plugin and set true in the port cloning "clone-remote-port: true" in the Geyser configuration. As an optional step, you can install the "Floodgate" plugin."
+    },
```
Comment (Konicai) [inline _data/providers.json:13]: It sounds like the `default` description template can be used instead - see the Physgun entry for an example

**Review by Konicai — CHANGES_REQUESTED** [review]

File: _data/providers.json (hunk 1 above)
Comment (JavituRec) [reply _data/providers.json:13]: Ready

**Review by JavituRec — COMMENTED** [review]
Modified

**Comment by Konicai** [comment]
keep the alphabetical sorting too please

File: _data/providers.json [hunk 2]
```diff
@@ -6,6 +6,11 @@
     "java_ip": "Make sure your remote address is 'auto', uncomment `bedrock address`, and change `0.0.0.0` to your Java server's IP."
   },
   "built_in": [
+    {
+      "name": "GameProHost",
```
Comment (onebeastchris) [inline _data/providers.json:10]: The list is made to be alphabetical, and is divided into three sections - Built-in Geyser, Support for Geyser, and Unsupported. Based on the description, please move your host entry to the "support for geyser" part & insert it to keep alphabetical order.

**Review by onebeastchris — CHANGES_REQUESTED** [review]
Other than that, looks good

File: _data/providers.json (hunk 2 above)
Comment (JavituRec) [reply _data/providers.json]: Thanks for the clarification, it's already solved.

**Review by JavituRec — COMMENTED** [review]
Solved

**Review by onebeastchris — APPROVED** [review]

**Pull request merged**